src/test/java/io/sealights/plugins/sealightsjenkins/buildsteps/cli/executors/SendExternalReportTest.java

```diff
@@ -80,7 +82,28 @@ private BaseCommandArguments createBaseCommandArguments() {
         baseCommandArguments.setBuildName("1");
         baseCommandArguments.setBranchName("branchy");
         baseCommandArguments.setEnvVars(new EnvVars());
+        baseCommandArguments.setLogConfiguration(new LogConfiguration());
         return baseCommandArguments;
     }
 
+    private List<String> createExpectedCommand(BaseCommandArguments baseCommandArguments) {
+        List<String> expected = new ArrayList<>();
+        expected.add("path/to/java");
+        expected.add(AbstractCommandExecutor.formatTagProp());
+        expected.addAll(baseCommandArguments.getLogConfiguration().toSystemProperties());
+        expected.add("-jar");
+        expected.add("agent.jar");
```

Comment:
This code is very similar in all of the tests.
Please refactor.

Reply:
Absolutely agree, but i need to do some hard refactoring (probably using a base class).
Dont think its part of this scope.
I opened  a ticket: https://sealights.atlassian.net/browse/SLDEV-7231